src/build_queue.rs

```diff
@@ -290,9 +295,10 @@ fn set_yanked(conn: &mut postgres::Client, name: &str, version: &str, yanked: bo
                 );
             } else {
                 debug!("{}-{} {}", name, version, activity);
-            }
+            };
+            Ok(())
         }
-        Err(err) => report_error(&err),
+        Err(err) => Err(err.into()),
```

Comment:
This can probably become `let rows = conn.execute(...)?;` now.

Reply:
I updated this while rebasing